src/commands/delifeq.json

```
@@ -0,0 +1,61 @@
+{
+    "DELIFEQ": {
+        "summary": "Delete key if value matches string.",
+        "complexity": "O(1)",
+        "group": "string",
+        "since": "9.0.0",
+        "arity": 3,
+        "function": "delifeqCommand",
+        "command_flags": [
+            "WRITE"
+        ],
+        "acl_categories": [
+            "KEYSPACE",
+            "STRING"
```

"KEYSPACE" seems to be used for commands that work on keys of all types, such as DEL, EXIST, EXPIRE, SCAN, etc.

I think it should be only "STRING" for this command.

```suggestion
            "STRING"
```